monkestation/code/modules/meteor_shield/meteor_shield_coverage.dm

```diff
@@ -0,0 +1,40 @@
+#define TRAIT_METEOR_SHIELD_FIELD_MONITORED	"meteor_shield_field_monitored"
```

Comment:
why isn't this in the trait file

Reply:
bc it's literally only used here, as a marker for this one element.